docs/configuration.md

```diff
@@ -135,6 +139,10 @@ of the most common options to set are:
     Having a high limit may cause out-of-memory errors in driver (depends on spark.driver.memory
     and memory overhead of objects in JVM). Setting a proper limit can protect the driver from
     out-of-memory errors.
+    
+	<br /><em>Note:</em> In client mode, this config must not be set through the <code>SparkConf</code>
+	directly in your application, because the driver JVM has already started at that point.
+	Instead, please set this through the default properties file.</td>
```

Comment:
this doesn't actually apply here. `maxResultSize` can be read after the JVM is started, so please remove this line here.